pkg/globalnet/controllers/cluster_egressip_controller.go

```diff
@@ -173,11 +171,9 @@ func (c *clusterGlobalEgressIPController) validate(numberOfIPs int, egressIP *su
 			Reason:  "ZeroInput",
 			Message: "No global IPs to allocate",
```

Comment:
```suggestion
			Message: "No global IPs to allocate",
```
The message could be confusing. Shall we use "GlobalIP not allocated"

Reply:
I think "GlobalIP not allocated" might convey we tried to allocate but couldn't. We trying to say the user has specified to  not allocate any so perhaps "The specified NumberOfIPs is 0"?

Reply:
Yes, "The specified NumberOfIPs is 0" looks good.

Reply:
Done